components/admin-panel/sections/ActivityLog/index.js

```diff
@@ -231,7 +231,7 @@ const ActivityLog = ({ accountSlug }) => {
       ) : loading ? (
         <LoadingPlaceholder width="100%" height={163} />
       ) : !data?.activities?.nodes ? (
-        <MessageBox type="error" withIcon>
+        <MessageBox type="error" withIcon title="You must be admin">
```

Comment:
This was a leftover from testing having a title prop in the MessageBox, but that was removed since we already define titles in the child of the component, thanks for catching!